site/en/docs/web-platform/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: 'layouts/project-landing.njk'
+title: 'Web Platform'
+description: 'Documentation for Web Platform APIs.'
+---
+
+This section contains documentation for Web Platform APIs. Some are currently in an experimental status in Chrome. For example, origin and developer trials, or features only found in Canary. Others have shipped in stable Chrome, however are unimplemented or currently under development in other engines.
```

Comment:
```suggestion
This section contains documentation for Web Platform APIs. Some are currently in an experimental status in Chromium. For example, origin and developer trials, or features only found in Canary. Others have shipped in stable Chromium, however are unimplemented or currently under development in other engines.
```

Reply:
Not sure about this one. We might suggest that not all Chromium-derivates enable an API if it's supported in the engine. Example: Brave chose to disable the File System Access API despite the engine supporting it.

Reply:
Yeah, I was thinking that this is really Chrome, or at least we can be sure they are available in Chrome.

With shipped things we should be clear about status, especially if we know that some Chromium-based browsers haven't shipped. So I think I'll leave this as Chrome based on that?

Reply:
Let's leave it at Chrome. That's one advantage of writing on DCC :-)